modules/firestore/README.md

```diff
@@ -0,0 +1,167 @@
+# Firestore
+
+This module allows to a fiewstore datatabase, fields, indexes and documents.
+
+## Examples
+
+### New database
+
+```hcl
+module "firestore" {
+  source     = "./fabric/modules/firestore"
+  project_id = "my-project"
+  database = {
+    name        = "my-database"
+    location_id = "nam5"
+    type        = "FIRESTORE_NATIVE"
+  }
+}
+# tftest modules=1 resources=1 inventory=new-database.yaml
+```
+
+### New database with weekly backup
+
+```hcl
+module "firestore" {
+  source     = "./fabric/modules/firestore"
+  project_id = "my-project"
+  database = {
+    name        = "my-database"
+    location_id = "nam5"
+    type        = "FIRESTORE_NATIVE"
+  }
+  backup_schedule = {
+    retention         = "86400s"
+    weekly_recurrenve = "MONDAY"
+  }
+}
+# tftest modules=1 resources=2 inventory=new-database-with-weekly-backup.yaml
+```
+
+### New database with document
+
+```hcl
+module "firestore" {
+  source     = "./fabric/modules/firestore"
+  project_id = "my-project"
+  database = {
+    name        = "my-database"
+    location_id = "nam5"
+    type        = "FIRESTORE_NATIVE"
+  }
+  documents = [{
+    collection  = "my-collection"
+    document_id = "d3db1c14-e56d-4597-af1c-f95c2d2290c1"
+    fields = {
+      field1 = "value1"
+      field2 = "value2"
+    }
+  }]
+}
+# tftest modules=1 resources=2 inventory=new-database-with-document.yaml
+```
+
+### Existing database with document
+
+```hcl
+module "firestore" {
+  source     = "./fabric/modules/firestore"
+  project_id = "my-project"
+  database = {
+    name = "my-database"
+  }
+  database_create = false
+  documents = [{
+    collection  = "my-collection"
+    document_id = "d3db1c14-e56d-4597-af1c-f95c2d2290c1"
+    fields = {
+      field1 = "value1"
+      field2 = "value2"
+    }
+  }]
+}
+# tftest modules=1 resources=1 inventory=existing-database-with-document.yaml
+```
+
+### New database with database field
+
+```hcl
+module "firestore" {
+  source     = "./fabric/modules/firestore"
+  project_id = "my-project"
+  database = {
+    name        = "my-database"
+    location_id = "name5"
+    type        = "FIRESTORE_NATIVE"
+  }
+  fields = [{
+    collection = "my-collection"
+    field      = "1"
+    indexes = [
```

Comment:
same comment here, would it make sense to use a map?

Reply:
done